### DISCUSSION TITLE
Repeatedly wrong values

### DISCUSSION THREAD

**Discussion by badrandolph** [jomjol/AI-on-the-edge-device#3409] [General] [answered]
Hi, 

I keep having the issue that numbers are repeatedly misinterpreted. In my opinion, the settings and the quality of the camera should be correct to ensure the values are read properly. I have already tested all models for analog.
Attached are a few screenshots.

![Screenshot 2024-11-27 103218](https://github.com/user-attachments/assets/361cbcb0-5234-4172-b156-ae701c434607)
![Screenshot_20241126-205941_Samsung Internet](https://github.com/user-attachments/assets/8479698d-3f85-42b2-9cf0-c78d2fd5df83)
![Screenshot_20241126-205950_Samsung Internet](https://github.com/user-attachments/assets/c90f078b-33ad-4782-a5e7-9b45fbb3c8ad)

Thank you for your help.

**Comment by caco3** [comment, marked as answer]
Your contrast is not good enough.
Also the digit ROIS are not set properly set. Please read the documentation! https://jomjol.github.io/AI-on-the-edge-device-docs/

**Comment by badrandolph** [comment]
Thanks, i try al lot more settings, with more contrast, now i have no more errors.
